tests/flow/test_params.py

```diff
@@ -149,3 +149,9 @@ def test_id_scan(self):
         plan = redis_graph.execution_plan(query, params=params)
         self.env.assertIn('NodeByIdSeek', plan)
 
+    def test_escaped_parameter_name(self):
+        params = {'`escaped:param`': 5}
```

Comment:
Did we checked if parameter name containing backticks is supported in Neo ?
I think we need to review the change to the parser aswell

Reply:
Backtick-escaping parameter names is not supported in the Bolt client, but is supported in `cypher-shell`:
```
neo4j@neo4j> :param `backtick:escape` => 5
neo4j@neo4j> RETURN $`backtick:escape`;
+--------------------+
| $`backtick:escape` |
+--------------------+
| 5                  |
+--------------------+

1 row available after 4 ms, consumed after another 0 ms
```
The parser change can be seen here:
https://github.com/RedisGraph/libcypher-parser/commit/9d1636d02fcbbe7078fb00a5df76020adf205f1e